includes/class-plugin.php

```diff
@@ -643,6 +645,83 @@ public function show_admin_notices() {
         }
     }
 
+    /**
+     * Display the admin bar menu item.
+     *
+     * @param \WP_Admin_Bar $wp_admin_bar
+     *
+     * @return void
+     */
+    public function render_admin_bar( $wp_admin_bar ) {
+        if ( defined( 'WP_REDIS_DISABLE_ADMINBAR' ) && WP_REDIS_DISABLE_ADMINBAR ) {
+            return;
+        }
+
+        if ( ! $this->current_user_can_manage_redis() ) {
+            return;
+        }
+
+        $wp_admin_bar->add_node(
+            [
+                'id'    => 'redis-cache',
+                'title' => __( 'Object Cache', 'redis-cache' ),
+            ]
+        );
+
+        if ( $this->get_redis_status() ) {
+            $wp_admin_bar->add_node(
+                [
+                    'parent' => 'redis-cache',
+                    'id' => 'redis-cache-flush',
+                    'title' => __( 'Flush Cache', 'redis-cache' ),
+                    'href' => $this->action_link( 'flush-cache' ),
```

Comment:
What happens when this is called on the frontend?

Reply:
The cache is flushed, and the user is redirected to the ROC settings screen.

Reply:
Let's count the days until someone complains about this 🤣